index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.css">
+    </link>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Serif:opsz@8..144&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <h1 class="header">John Doe </h1>
+        <img src="menu.svg" alt="menu" class="navbarmenu" />
+    </nav>
+    <main>
+        <section class="textsection">
+            <div>
+                <h2> Hey There. I’m Alex</h2>
+                <p class="developer">I’am a Software Developer</p>
+                <p class="build">I can help you build a product , feature or website Look through some of my work and
+                    experience! If you
+                    like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <div class="icons">
+                    <img src="Icon - Linkedin.svg" alt="linkedin" class="linkedin" />
+                    <img src="Icon - Angelist.svg" alt="angelist" class="angelist" />
+                    <img src="Icon - Medium.svg" alt="medium" class="medium" />
+                    <img src="Icon - Twitter.svg" alt="twitter" class="twitter" />
+                    <img src="Github.svg" alt="github" class="github" />
+                </div>
```

Comment:
- [ ] I would suggest you please wrap this social media icon with a ul tag to list them and for each icon use a li tag with the `a` tag.